examples/sample-app/version-2/app-pre-deploy-eval.yaml

```diff
@@ -8,4 +8,4 @@ spec:
     - keptnMetricRef:
         name: available-cpus
         namespace: podtato-kubectl
-      evaluationTarget: ">1"
+      evaluationTarget: "<1"
```

Comment:
why was this changed?

Reply:
because the example was not working

Reply:
having the requirement of `available-cpus` to be less than one seems very odd though - I will try out this example in my environment and see if it also fails here

Reply:
ok so for me it was failing with the `<1` criteria - given the meaning of that particular metric, I suggest to use `>1` for the evaluationTarget again, as this will likely confuse someone trying this out otherwise due to an evluation failing due to the number of available resources being too high